components/automate-cli/cmd/chef-automate/certRotate_test.go

```diff
@@ -775,70 +849,71 @@ func TestGetFrontEndIpsForSkippingCnAndRootCaPatching(t *testing.T) {
 		newRootCA           string
 		newCn               string
 		oldCn               string
+		oldRootCA           string
 		node                string
-		currentCertsInfo    *certShowCertificates
 		infra               *AutomateHAInfraDetails
-		skipIpsList         []string
+		skipIps             bool
```

Comment:
Give more meaning full name to skipIps variable. ex. ipSkipped,isIpSkip

Reply:
done